spec/models/organization_spec.rb

```diff
@@ -3,23 +3,31 @@
 require 'rails_helper'
 
 RSpec.describe Organization do
-  let(:organization) { create(:organization, business_plan_name: 'Editorial Basic') }
+  let!(:organization) { create(:organization, business_plan_name: 'Editorial Basic') }
 
   describe '#notify_admin' do
     subject { organization.update(params) }
 
+    let!(:admin) { create_list(:user, 3, admin: true) }
+    let!(:users) { create_list(:user, 2) }
+
     context 'does not schedule a PostmarkAdpater::Outbound job' do
       let(:params) { { name: 'SomethingElse', upgrade_discount: 100 } }
 
       it 'for other updates' do
-        expect { subject }.not_to have_enqueued_job
+        expect { subject }.not_to have_enqueued_job.on_queue('default').with(
```

Comment:
This looks dangerous. Can we please stay specific for what we expected *to be* there and unspecific for things that we expect **not** to be there?

Reply:
I agree